### ISSUE TITLE
[MSFT-104] Challenges page: Screen reader doesn't announce the type of element for the 'save filter' and 'clear filter' controls.

### ISSUE THREAD

**Issue by veshu** [importtest/ewe#71]
**Steps to Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 2. Ensure the screen reader is on. 3. Activate the filter. 4. Use the down arrow to understand the layout of the filter. 5. Notice that the screen reader announces 'save filter' and 'clear filter' instead of 'save filter button' and 'clear filter button'.

**Expected Result**
Screen reader should announce these controls as 'save filter button' and 'clear filter button'.

**Actual Result**
Screen reader announces 'save filter' and 'clear filter' instead of 'save filter button' and 'clear filter button'.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 4. Robust
**WCAG Level**: A
**WCAG Success Criterion**: 4.1.2 Name, Role, Value
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2010
